osf_pigeon/pigeon.py

```diff
@@ -157,8 +155,8 @@ async def get_metadata_for_ia_item(json_metadata):
         "osf_registration_schema": embeds["registration_schema"]["data"]["attributes"][
             "name"
         ],
-        "source": osf_url
-        + json_metadata["data"]["relationships"]["registered_from"]["data"]["id"],
+        "osf_project": json_metadata["data"]["relationships"]["registered_from"]["links"]["related"]["href"],
```

Comment:
should this be `html` instead of `href`?

Reply:
yep, https://api.staging.osf.io/v2/registrations/